Dockerfile

```diff
@@ -17,8 +17,6 @@ ADD . .
 
 RUN go get -v -t $(go list ./... | grep -v /vendor/)
 
-RUN go install -v ./buffalo
-
```

Comment:
Sure, but are you certain it's needed? The go get should be handling this. But I can add it back, no issue.